src/main/kotlin/step2/Operator.kt

```diff
@@ -0,0 +1,19 @@
+package step2
+
+enum class Operator(expr: String, evaluate: (num1: Int, num2: Int) -> Int) {
+    PLUS("+", { num1: Int, num2: Int -> num1 + num2 }),
```

Comment:
람다 표현식의 매개 변수 타입은 추론이 가능하면 생략할 수 있습니다.

```suggestion
    PLUS("+", { num1, num2 -> num1 + num2 }),
```

Reply:
JS하던 습관때문에 무의식적으로 타입선언을 계속하고 있었네요!
좋은 지적 감사합니다 :)